components/iam/pkg/server/configs.go

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) 2022 Gitpod GmbH. All rights reserved.
+// Licensed under the GNU Affero General Public License (AGPL).
+// See License-AGPL.txt in the project root for license information.
+
+package server
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+)
+
+type GoogleTestConfig struct {
```

Comment:
What's a GoogleTestConfig for?

Reply:
And could we make it GoogleConfig instead? If we need to change it then that's okay, but Test in the name is bit of a smell.

Reply:
`GoogleTestConfig` is a hard coded config set, to be removed in the following.

Reply:
Okay, make it `GoogleConfig`, possibly with a comment saying to be removed once we're reading configs from DB.

Reply:
adding comment